### ISSUE TITLE
Welcome screen links and text does not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51974]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Welcome screen.
3. Apply Light Visual Studio Theme.
4. Start Color Contrast Analyzer to determine the color contrast ratio of headings("Start","Recent","Help","Customize","Learn"),links(in blue) and content(within buttons on the right).

**Actual:**
Color Contrast ratio of the links,headings and content is less than 4.5:1 in Light VS theme.

**Expected:**
Color Contrast ratio of links,headings and content should be >=4.5:1 in Light VS theme.

**Recommendations:**
Adjust color of that element to meet the minimum color contrast requirement. 
Or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Low visioned users will not clearly understand the text(links,headings and content) written on the screen.

**MAS Reference:**
MAS 1.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment for Reference:**
![a11y_vscode_welcomescreen_colorcontrastanalyzer_1](https://user-images.githubusercontent.com/39204909/41457110-32646c48-70a0-11e8-9c82-07382f3b901a.png)
![a11y_vscode_welcomescreen_colorcontrastanalyzer_2](https://user-images.githubusercontent.com/39204909/41457119-3ab8455e-70a0-11e8-8f9f-a2a43f615f46.png)
![a11y_vscode_welcomescreen_colorcontrastanalyzer_3](https://user-images.githubusercontent.com/39204909/41457122-3e78c3bc-70a0-11e8-977e-65682369c542.png)
![a11y_vscode_welcomescreen_colorcontrastanalyzer_4](https://user-images.githubusercontent.com/39204909/41457123-42662c6c-70a0-11e8-92ce-c4cfbddf2b48.png)

Does this issue occur when all extensions are disabled?: Yes
